vllm/distributed/communication_op.py

```diff
@@ -23,6 +23,16 @@ def tensor_model_parallel_gather(input_: torch.Tensor,
     """Gather the input tensor across model parallel group."""
     return get_tp_group().gather(input_, dst, dim)
 
+def tensor_model_parallel_broadcast(input_: torch.Tensor,
+                                 src: int = 0,
+                                 ) -> torch.Tensor:
+    """Broadcast the input tensor across model parallel group."""
+    return get_tp_group().broadcast(input_, src)
+
+def model_parallel_broadcast_object_list(input_: List[Any],
```

Comment:
`tensor_model_parallel_broadcast_object_list` would be more consistent with other func names.